docs/components/clerk-provider.mdx

```diff
@@ -81,80 +85,17 @@ ReactDOM.render(
-    {
-      cells: [
-        <code>publishableKey</code>,
-        <code>string</code>,
-        <>Clerk publishable key for your instance. This can be found in your Clerk Dashboard.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>frontendApi</code>,
-        <code>string</code>,
-        <>The frontend API host for your instance. This can be found in your Clerk Dashboard.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>navigate?</code>,
-        <code>{`(to: string) => Promise<unknown> | void`}</code>,
-        <>A function which takes the destination path as an argument and performs a "push" navigation.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>clerkJSVariant?</code>,
-        <code>string</code>,
-        <>If your web application uses only Control components you can set this value to headless and load a minimal ClerkJS bundle for optimal page performance.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>supportEmail?</code>,
-        <code>string</code>,
-        <>Optional support email for display in authentication screens. Will only affect Clerk Components and not Account Portal pages.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>localization?</code>,
-        <code>object</code>,
-        <>Optional object to localize your Components. Will only affect Clerk Components and not Account Portal pages.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>appearance?</code>,
-        <code>object</code>,
-        <>Optional object to customize your Components. Will only affect Clerk Components and not Account Portal pages.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>allowedRedirectOrigins?</code>,
-        <code>{`Array<string | RegExp>`}</code>,
-        <>Optional array of domains used to validate against the query param of an auth redirect.
-        <br/>If no match is made, the redirect is considered unsafe and the default redirect will be used with a warning passed to the console.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>afterSignInUrl</code>,
-        <code>string</code>,
-        <>Full URL or path to navigate after successful sign in.</>,
-      ],
-    },
-    {
-      cells: [
-        <code>afterSignUpUrl</code>,
-        <code>string</code>,
-        <>Full URL or path to navigate after successful sign up.</>,
-      ],
-    }
-  ]}
-/>
+| Name                      | Type                                       | Description                                                                                                                                                                                                                |
+|---------------------------|--------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

```

Comment:
There's a lot of extra spacing in this table.. I'd probably refactor it to have normal spacing like the other tables.

```suggestion
| Name | Type | Description |
| --- | --- | --- |
```

Reply:
The way the table is formatted right now is how Prettier formats tables (see [this playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAJOBDAJnATgHSgF4TTDCB9AGQggGsAaCgAkh2YCMAbCMOgZ2YZccZgDMIuALYYY8LMwBUMWgEJF5KAAMdzAFb9CYgK5QwMAJbRmFnLAswAngAoAHgEpmwZiJjHcUMyuzAC+hDpamgAKFjwwAD4Agha4AA6SCagQ-oZQALR58UgFSPElhABSEAAWgQAiEHDxAMoA0gDi8QCMAMw9AByVGFCiAEqN8RUAYq3xPV0ALJp5zCtrqxvrW5ubmgDUzFQW-DCEzAcA7g7VQswA2lwWUHQAuszOAPQqH-wQUnAXap4ODuQgHYYKO7DCAwIG4ZjQOAvTRne7Q2F4BEjF5IZjMapyVJID4fOCuDBSVJcOAAOkgUmYAHIAKLkynU5iWGDUxmaGgiBkWVL8YwMrAQHjw-gOIT-GAMVjQfhwcxwPzw7BC45gJ4Ac2YcEeMBphAAwv4MBwHP5FVBlar1cwZK4LFJjIJcMd3QrlQpna73ZzDVxA5YzLZTDADcaQAwQBBUpYlchQMJcBALlFhAh+MgQBguBcMI5c3GOLgMHw1c0KXAjiNkGIC8qyxWqzBmqlK3rkDBcMY4HHlVILL3+4OQNKoLrqQBFYwwuCN5sTgyuZp6ucL+DLrgtkAAR23cCi6eFeYw-DyIzgOCwsZAfYwsT1pr+MgvXC4D6nM7giTkT0OGMeAojweslyQJs9wnAkpC4AB1aoHDgfguzAOBmhzBwLAANwcRw8zAfhSxAXCBwASSgOwOzAT1E0Sajmicald33VJ02VBCK1SPMONQvBcKXOMnmVXAYFPDBdQ-KCVzjLtcDEvMZFwOhxQuKAHw4p4YAQ2xYWQfoAAY4xEI8UhPCtpIwNiJxgS09KwAykAAJjjd04AAFUtXNZJguM4CkDhbzvKhhl1YwpLgKZJBkOQexQDAQIgEAQhCIA)). So once we introduce linting/formatting into our workflow the tables will be formatted like that normally.

You can configure it tho, see https://github.com/prettier/prettier-vscode/issues/762 for context

Reply:
Oh amazing!